lib/windows/launcher/components/AppIcon.jsx

```diff
@@ -75,10 +76,9 @@ const AppIcon = ({ app }) => {
             }}
         >
             <img
-                src={app.iconPath}
+                src={app.iconPath || `${app.url}.svg`}
```

Comment:
What happens if the .svg does not exist? I guess that is a valid scenario.

Reply:
The blank icon is displayed, and a 404 error is logged in the console.

Reply:
I don't like the idea of loading a resource from the net. I would like to see a feature to download locally to the folder of the source and load it from there.

Reply:
Ok. I'll work on that. But this opens up a can of worms. Like displaying of a lazy downloaded image. I'll probably update app.iconPath when a image bacomes available. Also the whole caching issue. I could just trigger a download every time. Or save the ETag somewhere and give that to the server. Or Trigger an when an app changes. Then I need to detect that. And decide on how to invalidate the downloaded image. Delete it? Just trigger the download and hope that it goes trough?